enderio-conduits/src/main/java/crazypants/enderio/conduits/conduit/liquid/EnderLiquidConduitNetwork.java

```diff
@@ -26,6 +28,7 @@
 
   List<NetworkTank> tanks = new ArrayList<NetworkTank>();
   Map<NetworkTankKey, NetworkTank> tankMap = new HashMap<NetworkTankKey, NetworkTank>();
+  Set<NetworkTankKey> hasMultipleTanksSet = new HashSet<NetworkTankKey>();
```

Comment:
Could this be a field of `NetworkTank` instead?

(Then the initialization from below would move to NetworkTank's constructor.)

Reply:
Good idea, implemented in 0a5e738